.github/workflows/pr-compile-check.yaml

```diff
@@ -30,3 +30,15 @@ jobs:
         push: false
         load: false
         provenance: false
+
+    - name: Attempt to build current source for CentOS 7 using static config
+      uses: docker/build-push-action@v5
+      with:
+        context: .
+        file: ./dockerfiles/Dockerfile.centos7
+        # No need to use after this so discard completely
+        push: false
+        load: false
+        provenance: false
+        build-args:
+          - EXTRA_BUILD_ARGS=-DFLB_STATIC_CONF=/tmp
```

Comment:
This may need quoting or similar, the linter does not like it anyway.
Using a dedicated variable for FLB_STATIC_CONF would make this better I think.